src/nifti/parseNiftiFile.js

```diff
@@ -36,7 +37,7 @@ export function parseNiftiHeader (fileData) {
     isDataInColors: isDataInColors(nifti, header.dims, header.datatypeCode)
   };
   const pixelSpacing = header.pixDims.slice(1, 4);
-  const orientationMatrix = getOrientationMatrix(header);
+  const orientationMatrix = _.cloneDeep(getOrientationMatrix(header));
```

Comment:
How nested is this object? Would it be straightforward to modify `getOrientationMatrix` to copy its object and array components instead of throwing it into `cloneDeep`?

Reply:
The orientationMatrix is just a 4x4 matrix only, and now update to remove the clonedeep usage and make copy "header.affine.map((row) => row.slice())"